Documentation/configuration/metrics.rst

```diff
@@ -38,7 +38,7 @@ Policy
 
 Events external to Cilium
 -------------------------
-* ``event_ts``: Last timestamp when we received an event. Further labelled by
+* ``event_ts``: Last timestamp when we received an event. Further labeled by
```

Comment:
I'm pretty sure this is a UK vs US thing. I imagine we decided to follow US spellings? If not, we should decide (to use the US ones despite them being wrong ;) )

Reply:
Yes, US spellings. I've also changed `behaviour` to `behavior`

Reply:
that's probably my fault. I'll work on my muscle memory.